include/constants/battle_config.h

```diff
@@ -275,4 +275,6 @@
 
 #define B_ENABLE_DEBUG                  TRUE     // If set to TRUE, enables a debug menu to use in battles by pressing the Select button.
 
+#define B_EXPANDED_MOVE_NAMES           FALSE    // If TRUE, move names are increased from 12 characters to 16 characters.
```

Comment:
Should be put in the "Interface settings" section, below `B_HIDE_HEALTHBOX_IN_ANIMS`.